authappliance/menu.py

```diff
@@ -455,7 +451,8 @@ def setup_tinc(self, local_vpn_ip, remote_vpn_ip, vpn_subnet, vpn_name='privacyi
         # Start the tinc nets
         start_command = 'tincd -n {}'.format(pipes.quote(vpn_name))
         # locally
-        proc = my_Popen(start_command, shell=True, stderr=PIPE)
+        proc = Popen(start_command, shell=True, stdout=PIPE, stderr=PIPE,
```

Comment:
Could this be something, that breaks tinc? The added ``stdout``?

Reply:
In case of a given `stdout=PIPE`, the process will be executed on a `communicate()` call. If the channels are `None` (default), the process will be executed immediately and since there is nothing to communicate, a simple `wait()` will finish the call.
If a channel is empty (stdout), the `communicate()`-call just returns an empty stdout.

Reply:
Done in c2ca84a